rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -37,8 +37,8 @@ local function balance()
 
   -- Round-Robin
   round_robin_lock:lock(backend_name .. ROUND_ROBIN_LOCK_KEY)
-  local index = round_robin_state:get(backend_name)
-  local index, endpoint = next(backend.endpoints, index)
+  local shd_idx = round_robin_state:get(backend_name)
```

Comment:
Can you rather call this `last_index`? That would be more descriptive IMHO.

Reply:
done.